README.md

```diff
@@ -130,24 +124,29 @@ task config
 
 The config file `st.config` contains all available configuration variables. Take a look at the descriptions provided in this file for details.
 
-## Build 
+## Build
 ```bash
-task image
+# build stboot disk image
+task disk
+# build stboot ISO image
+task iso
 ```
 
 ## Test
-``` bash 
+``` bash
 
-# run target installation
-task run
+# run stboot disk image
+task run-disk
+# run stboot ISO image
+task run-iso
 ```
 
 # OS-Package
 An OS package consists of an archive file (ZIP) and descriptor file (JSON). The archive contains the boot files (kernel, initramfs, etc.) and the descriptor file contains the signatures and other metadata.
 
 OS packages can be created and managed with the _stmgr_ tool. Source code of stmgr: https://github.com/system-transparency/stmgr/tree/main/tools/stmgr
 
-Once you have an OS kernel & initramfs containing the userspace and optionally a tboot kernel and appropriate ACM for TXT create an OS package out of it:
+Once you have an OS kernel & initramfs containing the user space, create an OS package out of it:
```

Comment:
Further up you wrote "userspace" and "user space". Not sure which of these should canonically be used but either way we should be consistent.

Reply:
let's stick with `userspace`